tests/syntax-tests/create_highlighted_versions.py

```diff
@@ -16,6 +16,16 @@
     "--italic-text=always",
 ]
 
+BINARY_OPTIONS = [
```

Comment:
maybe "BAT_OPTIONS"?

Reply:
Well, BAT_OPTIONS is what's used in your other, config setup. Have no qualms changing this to whatever you recommend.